packages/common/stateInfo.ts

```diff
@@ -32,6 +32,36 @@ export const isImplementedLocale = (l: Locale): l is Locale<ImplementedState> =>
 
 export type SignatureType = 'canvas' | 'upload'
 
+// To avoid type checking issues when calling this from client
+type StateSignatureType =
+  Record<ImplementedState, 'both' | SignatureType>
+  & Partial<Record<State, 'both' | SignatureType>>
+
+const stateSignatureType: StateSignatureType = {
+  'Arizona': 'both',
+  'Florida': 'both',
+  'Georgia': 'both',
+  'Maine': 'both',
+  'Maryland': 'both',
+  'Massachusetts': 'both',
+  'Michigan': 'both',
+  'Minnesota': 'both',
+  'Nebraska': 'upload',
+  'Nevada': 'both',
+  'New Hampshire': 'upload',
+  'New York': 'both',
+  'North Carolina': 'both',
+  'Oklahoma': 'both',
+  'Wisconsin': 'both',
+  'Wyoming': 'both',
+}
+
+// Uses State | undefined instead of ImplementedState to avoid type checking
+// issues when calling this function from client.
+export const eligibleSignatureType = (state: State | undefined) => {
```

Comment:
Can we use `ImplementedState` and on the client, throw an error if it's for the wrong state (i.e. `<Signature>` is only usable with a valid state.